homeassistant/components/comelit/cover.py

```diff
@@ -52,6 +51,7 @@ def __init__(
         self._api = coordinator.api
         self._device = device
         super().__init__(coordinator)
+        # Use config_entry.entry_id as base for unique_id because no serial number or mac is available
```

Comment:
Please break long strings around 88 characters per line.

Reply:
Here you go: https://github.com/home-assistant/core/pull/102783